cmds/fwtools/spidev/spidev_linux.go

```diff
@@ -69,10 +70,16 @@ func run(args []string, spiOpen spiOpenFunc, input io.Reader, output io.Writer)
 		return fmt.Errorf("%w:%v", errFlag, err)
 	}
 
-	if fs.NArg() != 1 {
-		return fmt.Errorf("%w: %s <raw|sfdp|id>", errCommand, fs.FlagUsages())
+	if fs.NArg() == 0 {
+		return fmt.Errorf("%w: %s <raw [bytes]...|sfdp|id>", errCommand, fs.FlagUsages())
```

Comment:
would it make sense to update lin 68 to be consistent with this new format ? 

line 68: `return fmt.Errorf("%w:<raw|sfdp|id>", errCommand)`

Reply:
ah, line 80 looks like matching  68, but mis-match with here. does it make sense to use one form to be consistent ?